### PR TITLE
feat(chronograf-cli): add TLS ciphers and versions

### PR DESCRIPTION
**Pull request by sranka**
https://github.com/influxdata/chronograf/pull/5591 introduces new chronograf cli options, so the docs herein

- [x] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Tests pass (no build errors)
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/chronograf/v1.8/tools/chronograf-cli/_index.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ chronograf [flags]
 |:---------|:------------------------------------------------------------|:--------------------|
 | `--cert` | File path to PEM-encoded public key certificate            | `$TLS_CERTIFICATE` |
 | `--key` | File path to private key associated with given certificate | `$TLS_PRIVATE_KEY` |
+| `--tls-ciphers` | Comma-separated list of cipher suites to use. Use 'help' cipher to print available ciphers. | `$TLS_CIPHERS` |
```
Comment (pierwill) [inline content/chronograf/v1.8/tools/chronograf-cli/_index.md:56]:
```suggestion
| `--tls-ciphers` | Comma-separated list of supported cipher suites. Use `help` to print available ciphers. | `$TLS_CIPHERS` |
```

**Review by pierwill — CHANGES_REQUESTED** [review]
A couple small changes.

**Comment by pierwill** [comment]
Thanks for this, @sranka!

**Comment by sranka** [comment]
@pierwill thank you for review, I applied your suggestions

**Comment by pierwill** [comment]
👍  Will hold for Chronograf 1.8.7 release.

**Review by pierwill — APPROVED** [review]

**Pull request merged**